jsconfig.json

```diff
@@ -1,5 +1,11 @@
 {
-  "compilerOptions": {
-    "experimentalDecorators": true
-  }
+  "compilerOptions": { "target": "es6", "experimentalDecorators": true },
+  "exclude": [
```

Comment:
why these changes? (just curious -- not familiar with jsconfig, or why we have it instead of tsconfig :sweat_smile: )